### ISSUE TITLE
Additional tab/page for list of configured/discovered clients for quick block/unblock

### ISSUE THREAD

**Issue by alexsannikov** [AdguardTeam/AdGuardHome#1808]
### Problem Description + Proposed Solution
I love very much the main page, where I can disable access to Internet for any device in just a one click. That is in the "most recent client" section, and only what I need to do is just to click "Block" or "Unblock" for chosen device/IP.
However, if client disappeared from the recent list, I need to go to "Settings" -> "Blocked IP" etc. and remove IP address from there manually.
Having all the devices in one place (excluding 127.0.0.1 of course) and ability to disable/enable Internet in just one click would be a perfect feature. I'd use it all the time for my kids :)

**Comment by DandelionSprout** [comment]
I know of a workaround, but sadly it'd require some trickery to have it disable internet for one kid while keeping it enabled for another kid.

Essentially, if one or more of the kids have been given the `user_child` tag in the client settings, adding `||*$ctag=user_child` to *Custom filtering rules* would've disabled internet for everyone whose IP addresses have been given that client tag.

To re-enable internet again with ~3 clicks, add a `!` in front of the entry.

**Comment by KizunaH** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/2150

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/AdGuardHome/issues/2075
